OWML.ModLoader/ModData.cs

```diff
@@ -34,30 +34,12 @@ public ModData(IModManifest manifest, IModConfig config, IModConfig defaultConfi
 
         public void UpdateSnapshot()
         {
-            if (Config != null)
-            {
-                _configSnapshot = Config.Copy();
-            }
-            else
-            {
-                _configSnapshot = DefaultConfig?.Copy();
-            }
+            _configSnapshot = Config != null ? Config.Copy() : DefaultConfig?.Copy();
         }
 
         public void ResetConfigToDefaults()
         {
-            if (DefaultConfig != null)
-            {
-                Config = DefaultConfig.Copy();
-            }
-            else if (Config != null)
-            {
-                Config.Enabled = true; //to preserve other settings
-            }
-            else
-            {
-                Config = new ModConfig();
-            }
+            Config = DefaultConfig != null ? DefaultConfig.Copy() : new ModConfig();
```

Comment:
WRONG 😝

Reply:
you forgot that it would delete all other settings

Reply:
that's the name of the method: ResetConfigToDefaults ? :)
if default config doesn't exist, resetting to new/empty config would give defaults that makes sense, at least to me.

Reply:
problem is if default-config doesn't exist and config has some non-standard settings it would make stuff super annoying

Reply:
and currently having default-config is not enforced 🤷‍♂️

Reply:
otherwise we might as well do FixConfigs differently to actually kinda enforce having default-config

Reply:
that is do
`if (DefaultConfig == null) DefaultConfig = new ModConfig();` at the start